src/main/java/com/laytonsmith/abstraction/bukkit/events/BukkitEntityEvents.java

```diff
@@ -415,6 +411,23 @@ public void setCancelled(boolean cancelled) {
 		public MCPlayer getPlayer() {
 			return new BukkitMCPlayer(e.getPlayer());
 		}
+
+		/**
+		 * @since Bukkit 1.9
+		 * @throws UnsupportedOperationException When this method does not exist in the current Bukkit version.
+		 */
+		@Override
+		public MCEquipmentSlot getHand() throws UnsupportedOperationException {
+			try {
+				if(e.getHand() == EquipmentSlot.HAND) {
+					return MCEquipmentSlot.WEAPON;
+				}
+				return MCEquipmentSlot.OFF_HAND;
+			} catch(NoSuchMethodError e) {
+				throw new UnsupportedOperationException(
+						"getHand() method does not exist in Bukkit version: " + Bukkit.getBukkitVersion());
```

Comment:
Multiple try-catches are used here to eventually return null data, even though the data technically exists even on old versions. It would be more efficient to check the version beforehand, and if the version is high enough check the method, otherwise just return WEAPON, since that's the hand it would be anyway.